config/initializers/face_to_face_holidays.rb

```diff
@@ -0,0 +1,11 @@
+CHRISTMAS_2016 = [
```

Comment:
This interim thing doesn't add anything. Just use one or the other. I think we can infer that this holiday is Christmas / New Year.